vars.tf

```diff
@@ -10,6 +11,16 @@ variable "aws_secret_key" {
   default = null
 }
 
+variable "app_env" {
+  description = "The abbreviated app environment (e.g. prod or stg)"
+  type        = string
+}
+
+variable "app_environment" {
```

Comment:
`app_environment` is  not used anywhere that I can see.

Reply:
`app_environment` is used to create the `name_tag_suffix` in `terraform-aws-tfc-backup-to-b2`.

Reply:
But in THIS module, you're not using `app_environment`.